modules/build/src/main/scala/scala/build/Builds.scala

```diff
@@ -14,6 +14,7 @@ final case class Builds(
     }
   def get(scope: Scope): Option[Build] =
     builds.find(_.scope == scope)
+  def isFail: Boolean = !builds.forall(_.success)
```

Comment:
```suggestion
  def anyFailed: Boolean = !builds.forall(_.success)
```
also, shouldn't we check all the builds here?
if so, then:
```suggestion
  def anyFailed: Boolean = !all.forall(_.success)
```

Reply:
good catch! thanks